cogs5e/models/character.py

```diff
@@ -280,6 +321,22 @@ async def set_active(self, ctx):
             {"owner": str(ctx.author.id), "upstream": self._upstream},
             {"$set": {"active": True}}
         )
+        
+    async def set_server(self, ctx):
+        """Sets the active character on the current server."""
+        guild_id = (str(ctx.guild.id) if ctx.guild is not None else None)
```

Comment:
Use `if ctx.guild is None: return` here, and document the behaviour when run in a non-guild context. It may be worth raising an exception instead of simply doing nothing, but I think just returning early is reasonable.